src/sentry/monitors/utils.py

```diff
@@ -78,6 +82,11 @@ def valid_duration(duration: Optional[int]) -> bool:
     return True
 
 
+# Used when creating check-ins or updating them after the config changes
+def get_checkin_margin_timedelta(checkin_margin: Optional[int]) -> timedelta:
+    return timedelta(minutes=int(checkin_margin or DEFAULT_CHECKIN_MARGIN))
```

Comment:
It's used in other places too no? I don't think we need the comment here, we can just look for the usages if we want to know.

I think it would be good have some kind of docstring on the function itself

```suggestion
def get_checkin_margin(checkin_margin: Optional[int]) -> timedelta:
    """
    Computes a timedelta given the checkin_margin (missed margin).
    If provided as an empty value will use the DEFAULT_CHECKIN_MARGIN
    """
    return timedelta(minutes=int(checkin_margin or DEFAULT_CHECKIN_MARGIN))
```

I think we could also drop the `_timedelta` since it's already on the method signature.

Reply:
yeah I added it for naming consistency with the other method `get_timeout_at_timedelta`

Reply:
I would probably just name that one `get_max_runtime`